libs/agno/agno/models/aws/bedrock.py

```diff
@@ -377,6 +377,11 @@ def invoke(
             if run_response and run_response.metrics:
                 run_response.metrics.set_time_to_first_token()
```

Comment:
This is weird, we should actually just do it on the assistant message and have that bubble up to the run response later